### DISCUSSION TITLE
Saving scrollback buffer to file

### DISCUSSION THREAD

**Discussion by lmcthbe** [wezterm/wezterm#5785] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

How to save the scrollback buffer to file for each open tab when exiting wezterm and get these history log files back into the scrollback buffer when  starting wezterm ?

**Comment by msooseth** [comment]
Actually, this is a really-really useful feature in my experience. I think it'd be nice to have a way of doing this in wezterm!

**Comment by ChanceHarrison** [comment]
[The docs](https://wezfurlong.org/wezterm/config/lua/wezterm/on.html#example-opening-whole-scrollback-in-vim) show an example of how this can be done (also see https://github.com/wez/wezterm/issues/222).
